audio/descriptors.c

```diff
@@ -1,22 +1,24 @@
-/**
- * Copyright (C) 2009 Ubixum, Inc. 
- * Copyright (C) 2014 Tim 'mithro' Ansell
- * Copyright (C) 2017 Kyle Robbertze
- *
- * This library is free software; you can redistribute it and/or
- * modify it under the terms of the GNU Lesser General Public
- * License as published by the Free Software Foundation; either
- * version 2.1 of the License, or (at your option) any later version.
- *
- * This library is distributed in the hope that it will be useful,
- * but WITHOUT ANY WARRANTY; without even the implied warranty of
- * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
- * Lesser General Public License for more details.
- *
- * You should have received a copy of the GNU Lesser General Public
- * License along with this library; if not, write to the Free Software
- * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
- **/
+ //Copyright (C) 2009 Ubixum, Inc. 
+ //Copyright (C) 2014 Tim 'mithro' Ansell
+ //Copyright (C) 2017 Kyle Robbertze
+//
+// This library is free software; you can redistribute it and/or
+// modify it under the terms of the GNU Lesser General Public
+// License as published by the Free Software Foundation; either
+// version 2.1 of the License, or (at your option) any later version.
+//
+// This library is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+// Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public
+// License along with this library; if not, write to the Free Software
+// Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+
+/** \file descriptors.c
+ * Describes the device according to Section 4 of the USB Audio Spec
```

Comment:
What type of device does it describe and what features does it have?

Reply:
An audio device. Section 4 is simple the descriptors section